doc/admin/pool_operations.md

```diff
@@ -162,7 +154,7 @@ To replace a pool's ACL with a new ACL:
 $ dmg pool overwrite-acl --pool <UUID> --acl-file <path>
 ```
 
-#### Updating entries in an existing ACL
+#### Updating an ACE
```

Comment:
More accurate would be "Adding and updating ACEs" since update-acl allows us to update more than one ACE at a time.